static/js/main.js

```diff
@@ -51,6 +52,7 @@ function refresh(){
     After, a check needs to be done with existing table elements to see if they were all updated.
     If they weren't, gray out the table box.  If they were, update the data, if new nodes exist add them.
     */
+    newxBeeArray =  getNodes('https://localhost:8000/api/list')
```

Comment:
You should not use absolute addresses. Using just `/api/list` works the same, and will not break when the address changes.